mule-extensions-api/src/main/java/org/mule/runtime/extension/api/loader/ExtensionModelLoader.java

```diff
@@ -88,7 +88,7 @@ public final ExtensionModel loadExtensionModel(ClassLoader pluginClassLoader, Ds
    * @throws IllegalArgumentException if there are missing entries in {@code attributes} or their type does not match the expected
    *                                  one.
    */
-  public final ExtensionModel loadExtensionModel(ExtensionModelLoadingRequest request) {
```

Comment:
look at comment in the other PR. There are pieces of this that you cannot offer to skip

Reply:
Reverted back to [c33d69b](https://github.com/mulesoft/mule-extensions-api/pull/922/commits/c33d69b703afeabe425834ee64baad85ea963a4b) version but with a different name for the overridable method as suggested.